lib/mix/tasks/hex.outdated.ex

```diff
@@ -215,7 +215,7 @@ defmodule Mix.Tasks.Hex.Outdated do
         end)
       end
 
-    List.last(versions)
+    Enum.max_by(versions, &Hex.Version.parse!/1)
```

Comment:
```suggestion
    Enum.max_by(versions, &Hex.Version.parse!/1, Hex.Version)
```

This will use the underlying comparator

Reply:
unfortunately this would only work on Elixir v1.10+ but we need to support v1.0+. As mentioned in the comments, I think we'll go with fixing it in minirepo instead. Thank you for this patch and reporting this bug.